ontology/yaml/resources/SAFETY/entity_types/LDS.yaml

```diff
@@ -68,6 +68,16 @@ LDS_NHDET:
   - LDS
   - NHDET
 
+LDS_H2M:
```

Comment:
This seems to break precedent for the LDS types, all other types are implement `alarm` fields, is this truly a H2 sensor? or is it an alarm?